codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/testutil/Rust.kt

```diff
@@ -97,6 +97,7 @@ object TestWorkspace {
                 mapOf(
                     "workspace" to
                         mapOf(
+                            "resolver" to "2",
```

Comment:
We didn't use to specify this so we'd get the warning for "please use resolver = '2'" when generating tests. This fixes that.